tests/testthat/test-db-integration.R

```diff
@@ -0,0 +1,81 @@
+test_that("can generate from db", {
+  con <- DBI::dbConnect(RPostgres::Postgres(),
+                        dbname = "northwind",
+                        host = "localhost",
+                        port = 5432,
+                        password = "northwind",
+                        user = "northwind")
+
+  db <- fb_generate(con, path = tempfile(), schema_name = "public")
+  fake_categories <- db$categories(category_id = 1L, category_name = "test cat")
+  expect_equal(fake_categories$category_id, 1L)
+  expect_equal(fake_categories$category_name, "test cat")
+  expect_true(is.na(fake_categories$description))
+  expect_true(is.na(fake_categories$picture))
+
+  fake_region <- db$region(region_id = c(1L, 2L), region_description = c("region 1", "region 2"))
+  expect_equal(fake_region$region_id, c(1L, 2L))
+  expect_equal(fake_region$region_description, c("region 1", "region 2"))
+})
+
+test_that("can generate and load from package", {
+  con <- DBI::dbConnect(RPostgres::Postgres(),
+                        dbname = "northwind",
+                        host = "localhost",
+                        port = 5432,
+                        password = "northwind",
+                        user = "northwind")
+
+  package_path <- system.file(package = "fakerbase")
+  generated <- fb_generate(con, package_path = package_path, schema_name = "public")
+
+  # devtools system.file shim only looks in package root
+  # so copy inst folder files into top level of package
```

Comment:
I'm not quite convinced that generating into a package makes much sense as opposed to just using absolute paths all the time, but I guess we'll see when we come to use it.